aws-lc-rs/src/aead.rs

```diff
@@ -360,7 +360,6 @@ fn open_within<'in_out, A: AsRef<[u8]>>(
 /// Intentionally not `Clone` or `Copy` since cloning would allow duplication
 /// of the nonce sequence.
 ///
-/// # FIPS
 /// Prefer [`RandomizedNonceKey`] for sealing operations.
```

Comment:
Should we just convert this line to a comment instead of deleting it? Should  the documentation indicate a preference for `RandomizedNonceKey` regardless?

Reply:
I was leaning towards making that the preferred type regardless of FIPS. Just seems like the easier API to use without throwing the complexity of NonceSequence and counters, and general problems of having to worry about avoiding nonce re-usage.